lib/ecstatic.js

```diff
@@ -57,27 +57,39 @@ var ecstatic = module.exports = function (dir, options) {
     catch (err) {
       return status[400](res, next, { error: err });
     }
-
-    var file = path.normalize(
-          path.join(root,
-            path.relative(
-              path.join('/', baseDir),
-              pathname
+    
+    // try and serve assets
+    var assetsDirectory = '_ecstatic-assets'
```

Comment:
If it only serves up icons and you can opt out (or in), we could just as soon put this in `/icons`